src/app/explore/explore.component.ts

```diff
@@ -357,6 +357,7 @@ export class ExploreComponent implements AfterViewChecked, OnDestroy, OnInit {
   }
 
   private loadMoreForCurrentSearch() {
+    console.log("in loadmoreForCurrentSearch");
```

Comment:
```suggestion
```
Unless this is useful for verifying in Staging for now?

Reply:
No, left in by mistake.